docs/batch-api.md

```diff
@@ -1,150 +1,150 @@
-  delegate: 'tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb',
-});
-```
-
-#### - The `withContractCall` method
-
-This method may be one of the most useful ones as it allows you to batch and emit multiple contract calls under one transaction. The parameter is also pretty simple: it takes the function you would call on the contract abstraction object if you would send a single transaction.
-
-```js
-const batch = await Tezos.batch()
-  .withContractCall(contract.methods.interact('tezos'))
-  .withContractCall(contract.methods.wait([['unit']]));
-```
-
-#### - The `with` method
-
-If you prefer having an array that contains objects with the different transactions you want to emit, you can use the `with` method. It allows you to group transactions as objects instead of concatenating function calls. The object you use expects the same properties as the parameter of the corresponding method with an additional `kind` property that indicates the kind of transaction you want to emit (a handy `opKind` enum is [exported from the Taquito package](https://github.com/ecadlabs/taquito/blob/master/packages/taquito-rpc/src/opkind.ts) with the valid values for the `kind` property).
-
-```js
-import { OpKind } from '@taquito/taquito';
-
-const batch = await Tezos.batch([
-  {
-    kind: OpKind.TRANSACTION,
-    to: 'tz1ZfrERcALBwmAqwonRXYVQBDT9BjNjBHJu',
-    amount: 2000000,
-    mutez: true,
-  },
-  {
-    kind: OpKind.ORIGINATION,
-    balance: '1',
-    code: validCode,
-    storage: 0,
-  },
-  {
-    kind: OpKind.DELEGATION,
-    delegate: 'tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb',
-  },
-]);
-```
-
-> Note: you cannot make contract calls with this method.
-
-#### - The `send` method
-
-After batching all the necessary operations together, you must use the `send` method to emit them. This step is very similar to what you would do to emit a single transaction.
-
-```js
-const batch = await Tezos.batch();
-/*
- * Here happens all the operation batching
- */
-const batchOp = await batch.send();
-console.log('Operation hash:', batchOp.hash);
-await batchOp.confirmation();
-```
-
-Like with other operations created by Taquito, the `send` method is a promise that returns an object where the operation hash is available under the `hash` property and where you can wait for the `confirmation` method to confirm the transaction (taking as a parameter the number of confirmations you would like to receive).
-
-## What are the limitations?
-
-The limitations of batched operations are within the constraints of single processes. For example, the gas limit of the Tezos blockchain limits the number of functions that can batch together.
-In addition to that, only a single account can sign batched operations.
-
-## References
-
-- [Integration tests](https://github.com/ecadlabs/taquito/blob/master/integration-tests/batch-api.spec.ts)
-- [Documentation](https://tezostaquito.io/typedoc/classes/_taquito_taquito.walletoperationbatch-2.html)
+---
```

Comment:
Did the line-endings (Windows v Unix?) change in this file? The whole file appears as changed, but I don't see any specific non-whitespace changes in here.

Reply:
I likely messed that up trying to get Grammarly to work. It was the first file I did. I'll do it again from scratch.

Reply:
Done.